django_examples/api/views.py

```diff
@@ -32,17 +32,20 @@ def upload_files(request):
 def upload_files_validation(request):
 
     def validation(filePath, mimetype):
+        with wand.image.Image(filename=filePath) as img:
+            if img.width != img.height:
+                return False
         size = os.path.getsize(filePath)
         if size > 10 * 1024 * 1024:
             return False
         return True
 
     options = {
-        'fieldname': 'myFile',
```

Comment:
@singhmonaaccolite What is reason to change fieldname ?

Reply:
We are addressing image here so the field name was changed, as the third editor checks if image height and width is equal or not.